### PR TITLE
Week 3 Build

### PR DESCRIPTION
**Pull request by saikrishnanooka-vl**

### CODE REVIEW

File: summit_blood_samples/blood_sample/migrations/0001_initial.py [hunk 1]
```diff
@@ -18,8 +18,9 @@ class Migration(migrations.Migration):
             name='ManifestImports',
             fields=[
                 ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
-                ('FilePath', models.CharField(max_length=255)),
-                ('CreatedAt', models.DateTimeField(auto_now_add=True)),
+                ('FilePath', models.CharField(max_length=500)),
```
Comment (jonc125) [inline summit_blood_samples/blood_sample/migrations/0001_initial.py:21]: Squashing the initial migration is fine for now. If we get to the stage with a production deployment that needs to maintain data across releases, I'll let you know!

File: summit_blood_samples/blood_sample/models.py [hunk 2]
```diff
@@ -1,65 +1,79 @@
...
 
 
 class BloodSampleImport(models.Model):
-    FilePath = models.CharField(max_length=255)
+    FilePath = models.CharField(max_length=500)
+    OriginalFileName = models.CharField(max_length=500)
     CreatedBy = models.ForeignKey(
         User, on_delete=models.CASCADE, related_name='BloodSampleCreatedBy')
-    CreatedAt = models.DateTimeField(auto_now_add=True, editable=False)
+    CreatedAt = models.DateTimeField()
```
Comment (jonc125) [inline summit_blood_samples/blood_sample/models.py:11]: Why was this changed?

File: summit_blood_samples/blood_sample/models.py [hunk 3]
```diff
@@ -1,65 +1,79 @@
...
+    Field = models.CharField(max_length=50)
+    FromValue = models.CharField(max_length=5000)
+    ChangedBy = models.ForeignKey(
+        User, on_delete=models.CASCADE, related_name='BloodSampleChangedBy')
+    ChangedAt = models.DateTimeField(auto_now=True, editable=False)
+
+    class Meta:
+        pass
+
+    def __str__(self):
```
Comment (jonc125) [inline summit_blood_samples/blood_sample/models.py:67]: Depending on where/if this is used, I'd expect something with a little more info! But it may just be for debugging? A comment or docstring stating this would be helpful. Similarly elsewhere.

File: summit_blood_samples/indigo/templates/indigo/base.html [hunk 4]
```diff
@@ -58,6 +58,9 @@
 <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js" integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo" crossorigin="anonymous"></script>
 <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/js/bootstrap.min.js" integrity="sha384-OgVRvuATP1z7JjHLkuOU7Xw704+h835Lr+6QL9UvYjZE3Ipu6Tp75j7Bh/kR0JKI" crossorigin="anonymous"></script>
 
+<!--TWBS Paginator-->
+	<script src="https://cdnjs.cloudflare.com/ajax/libs/twbs-pagination/1.4.2/jquery.twbsPagination.js"></script>
+	<script src="https://cdnjs.cloudflare.com/ajax/libs/twbs-pagination/1.4.2/jquery.twbsPagination.min.js"></script>
```
Comment (jonc125) [inline summit_blood_samples/indigo/templates/indigo/base.html:63]: Why do you have both min and non-min here?

File: summit_blood_samples/blood_sample/templates/upload-receipt.html [hunk 5]
```diff
@@ -10,29 +10,27 @@ <h3>1-Upload Receipt</h3>
...
+        <button type="button" class="btn btn-submit" id="uploadReceipt">Next</button>
     </div>
 </div>
-<style>
 
-
-</style>
 <script>
-$('body').delegate('#upload', 'click',function() {
+$('#uploadReceipt').click(function() {
```
Comment (jonc125) [inline summit_blood_samples/blood_sample/templates/upload-receipt.html:18]: I don't like the mix of naming conventions here (and elsewhere). Can we be consistent please?

**Review by jonc125 — APPROVED** [review]
I have managed to deploy locally using local.yml, although the instructions are rather opaque and I experienced several errors. I'll open separate issues to address those rather than holding up this PR.

I also took a brief look through the code; definitely not comprehensive, and I haven't looked at the templates or views yet, but I've noted a few thoughts. The main general comment would be that there are lots of commented out code lines, and a lack of docstrings or informative comments. I'd like to see more of the latter and less of the former :)

**Comment by saikrishnanooka-vl** [comment]
Hi @jonc125 This is not the final version, We will be giving final version in a couple of weeks with full docs and code formatting. However, this can be used to deploy.

**Comment by jonc125** [comment]
I'm aware it's still a work in progress. Just wanted to make sure these aspects were in the task list!

**Comment by saikrishnanooka-vl** [comment]
> I'm aware it's still a work in progress. Just wanted to make sure these aspects were in the task list!

Noted @jonc125 Thank you!
